{
  try {
    runTestCases(null,false);
  }
 catch (  Exception ex) {
    ex.printStackTrace();
    System.exit(-1);
  }
}
